nntrainer/utils/node_exporter.cpp

```diff
@@ -197,7 +195,10 @@ void Exporter::saveTflResult(
   auto strides = std::get<std::array<props::Stride, CONV2D_DIM>>(props);
   assert(strides.size() == CONV2D_DIM);
   auto padding = std::get<props::Padding2D>(props).get();
-  assert(padding == "same" || padding == "valid");
+  if (padding != "same" && padding != "valid") {
+    throw std::runtime_error("Unsupported padding type");
```

Comment:
Give more info for the error. E.g., 
```
        std::ostringstream ss;
        ss << "Unsupported padding type; \"" << padding << "\" is not supported. Use \"same\" or \"valid\".";
        throw std::runtime_error(ss.str());
    }
```

Reply:
If you have thrown an error, do you still need to do assert?

Reply:
Thank you for Review, Fix!!